provider/resources.go

```diff
@@ -5862,4 +5862,10 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 	prov.Resources["aws_iam_user_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
 		return attr(state, "userName"), nil
 	}
+	prov.Resources["aws_backup_restore_testing_plan"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
+		return attr(state, "name"), nil
+	}
+	prov.Resources["aws_backup_restore_testing_selection"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
+		return attr(state, "restoreTestingPlanName", "name"), nil
```

Comment:
Isn't this the other way around? i.e. `name:restore_testing_plan_name`

Reply:
I.e. that's how terraform defines the ID: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/backup_restore_testing_selection#import